src/server/http.rs

```diff
@@ -637,7 +637,7 @@ where
 
     fn shutdown(&self, force: bool) {
         if force {
-            self.settings.head().traverse::<TcpStream, H>();
+            self.settings.head().traverse(|ch: &mut HttpChannel<TcpStream, H>| ch.shutdown());
```

Comment:
I only added the type because it wouldn't compile without it.  I am new to rust, so maybe I am missing something in the definition of the traverse method?

Reply:
Ah sorry then, I was expecting that Rust would be able to see the type here(since it should see it from `self.settings.head()`), but if not. My  bad, I mistook it.